Remotion/ObjectBinding/Web.UnitTests/UI/Controls/BocBooleanValueImplementation/Rendering/BocCheckBoxRendererTest.cs

```diff
@@ -353,12 +351,6 @@ private void CheckImage (bool value, XmlNode outerSpan, string altText)
       Html.AssertAttribute (image, "alt", altText);
     }
 
-    private void CheckCheckBoxVisualizer (XmlNode outerSpan)
```

Comment:
Hmm... we can (should?) keep the check and continue to assert the span. It's just the css class that's done.